### PR TITLE
chore (readme.md):integrate travis-ci badge into readme.md

### PR DESCRIPTION
**Pull request by MemunaHaruna**
Start #144937441

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+MONGOHQ_URL=mongodb://localhost/kiba-cfh
```
Comment (q-ode) [inline .env:1]: Please untrack this file. It contains sensitive data and shouldn't be committed to the repo.

File: server.js [hunk 2]
```diff
@@ -6,6 +6,7 @@ var express = require('express'),
     passport = require('passport'),
     logger = require('mean-logger'),
     io = require('socket.io');
+require('dotenv').config();
```
Comment (q-ode) [inline server.js:9]: Nitpick: Let's move this to the top of the file.

Also, it's specified as a `devDependency`. What happens in production?

**Review by q-ode — CHANGES_REQUESTED** [review]
Great Job raising the first PR.

I see you've tried to follow convention in a couple of places, let's try to adhere across board.

File: .env (hunk 1 above)
Comment (MemunaHaruna) [reply .env:1]: Alright, Noted.

**Pull request closed without merging**